smap_l2_gridder/grid.py

```diff
@@ -29,23 +35,25 @@ def process_input(in_data: DataTree, output_file: Path, logger: None | Logger =
     """Process input file to generate gridded output file."""
     out_data = DataTree()
 
+    short_name = get_collection_shortname(in_data)
```

Comment:
(Putting this as close to the function signature for `process_input` as possible)

`pylint` reckons `logger` is an unused argument above. Maybe that can be cut from the function signature?

```
************* Module smap_l2_gridder.grid
smap_l2_gridder/grid.py:34:56: W0613: Unused argument 'logger' (unused-argument)
```

Reply:
Yes, I saw that, I left it because I've used the logger in other services, I just haddn't used it yet.  I guess I can remember it's easy to add back.

Reply:
d42750c